src/features/components/templates/signup/with-banner/social/index.tsx

```diff
@@ -51,6 +53,12 @@ const social_button_content: SocialButtonContent[] = [
 
 const PublicSignupSocial = () => {
     const handleSocialSignup = (data_provider: TSocialProvider) => {
+        Analytics?.trackEvent('ce_virtual_signup_form', {
+            signup_provider: data_provider,
+            action: 'started',
+            form_source: document.referrer,
```

Comment:
Maybe we need to check this not sure 🤔 
```suggestion
            form_source: document?.referrer,
```

Reply:
No @niloofar-deriv , this will always going to be there since it's a browser thing, no need to add a optional chain

Reply:
Ah ok then thanks for clarification 🙏